src/NuGet.Server.V2/Controllers/NuGetODataController.cs

```diff
@@ -0,0 +1,315 @@
+﻿using NuGet.Server.Core.DataServices;
+using NuGet.Server.Core.Infrastructure;
+using NuGet.Server.V2.OData;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.OData;
+using System.Web.Http.OData.Query;
+
+namespace NuGet.Server.V2.Controllers
+{
+    [NuGetODataControllerConfiguration]
+    public abstract class NuGetODataController : ODataController
```

Comment:
I have already ported (copied) the Conventions from NuGetGallery. I only included the conventions I actually could see the need for. NuGetGallery also includes conventions for some count operations that I havent yet understood why they are needed.

Just updated the top level comments with the rest of the comment as two tasks. See the tasks to check if I understood you correct.